src/manager.rs

```diff
@@ -536,10 +564,18 @@ impl<C: Store> Manager<C, Registered> {
             .expect("Time went backwards")
             .as_millis() as u64;
 
+        // start waiting for the contacts sync
+        info!("waiting for contacts sync for 3 minutes");
```

Comment:
Where are the three minutes coming from? I see no such mention in the code.

Reply:
True! Just a comment from something I ended up not doing, since I didn't want to introduce a dependency to `tokio`.

Reply:
`presage` already has an indirect dependency on `tokio` in `libsignal-service-hyper`.

Reply:
True, I'll do it then as it aligns with the goals of the project.